modules/juce_core/system/juce_PlatformDefs.h

```diff
@@ -171,7 +171,7 @@ namespace juce
   #if JUCE_LOG_ASSERTIONS
    #define jassert(expression)      JUCE_BLOCK_WITH_FORCED_SEMICOLON (if (! (expression)) jassertfalse;)
   #else
-   #define jassert(expression)      JUCE_BLOCK_WITH_FORCED_SEMICOLON ( ; )
+   #define jassert(expression)      JUCE_BLOCK_WITH_FORCED_SEMICOLON ( [[maybe_unused]] constexpr bool jassert_unused = false && (expression); )

```

Comment:
Is there a reason to prefer `[[maybe_unused]]` over `juce::ignoreUnused` (which has wider C++ support)?